sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_by_default_browser_profile_age_v1/query.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+  submission_date,
+  is_default_browser,
+  CASE
+    WHEN days_since_first_seen < 28
+      THEN TRUE
+    ELSE FALSE
+  END AS is_new_profile,
+  normalized_os_version,
+  os,
+  normalized_channel,
+  COUNT(*) AS row_count,
+FROM
+  `moz-fx-data-shared-prod.telemetry.clients_last_seen`
+WHERE
+  days_since_seen = 0
```

Comment:
I wonder if we should be using clients_daily instead? It seems to have all the columns that we need.

Reply:
telemetry.clients_daily. It's a much smaller table (just clients active on the day instead of any time in the previous 28 days), so it should be cheaper to query.
I asked Leif if there was any reason to not use it since he wrote the original query, and he said we could probably switch and maybe the columns weren't there when he wrote the query 3 years ago.